src/services/AuthService.ts

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+import jwt from 'jsonwebtoken';
+import { cache } from 'memory-cache';
+
+var JWT_SECRET = 'my-secret-key';
```

Comment:
the secret should not be in code. can we make sure we are storing the secrets in our secrets store? (can be a .env or a secret store on our cloud provider)